TKeazirian.HTTPServer.Tests/helpers/HelperFunctions.cs

```diff
@@ -13,4 +13,14 @@ public static string FormatTestRequest(string verb, string path, string body = "
                              $"{body}";
         return testRequest;
     }
+
+    public static string FormatTestGetRequest(string verb, string path)
+    {
+        string NewLine = "\r\n";
```

Comment:
This new line is in multiple places, and well as multiple files.  If the definition of what a `NewLine` is were to change, you would have to change it all over your code.